src/components/container/BloodPressureVisualization/BloodPressureVisualization.stories.tsx

```diff
@@ -29,7 +32,8 @@ const bpSurveyParams: SurveyBloodPressureDataParameters = {
 export const Default = {
 	args: {
 		previewState: "Default",
-		surveyDataSource: bpSurveyParams
+		surveyDataSource: bpSurveyParams,
+		weekStartsOn : "Monday"
```

Comment:
I think you can remove this.  The `weekStartsOn` value will always be ignored since a `DateRangeCoordinator` is being used in the `render` function.